fdbserver/DataDistribution.actor.cpp

```diff
@@ -1866,11 +1868,20 @@ ACTOR Future<Void> fetchBulkLoadTaskManifestEntryMap(Reference<DataDistributor>
 		// At this point, we have the global job manifest file stored locally at localJobManifestFilePath.
 		// This job manifest file stores all remote manifest filepath per range.
 		// Here, we want to get all manifest entries of the file with in the range specified by jobRange.
-		wait(getBulkLoadJobFileManifestEntryFromJobManifestFile(
-		    localJobManifestFilePath,
-		    jobRange,
-		    self->ddId,
-		    /*output=*/self->bulkLoadJobManager.get().manifestEntryMap));
+		wait(store(manifestMapRange,
+		           getBulkLoadJobFileManifestEntryFromJobManifestFile(
+		               localJobManifestFilePath,
+		               jobRange,
+		               self->ddId,
+		               /*output=*/self->bulkLoadJobManager.get().manifestEntryMap)));
+		// It is possible that the bulkload job is using a data set that does not entirely contain the bulkload job
+		// range. In this case, we give the bulkload job.
```

Comment:
I think this comment needs an edit. You throw exception in this case it looks like.